code/game/objects/effects/spawners/windowspawner.dm

```diff
@@ -6,7 +6,7 @@
 	var/useGrille = 1
 	var/windowtospawn = /obj/structure/window/basic
 
-/obj/effect/spawner/window/New()
+/obj/effect/spawner/window/initialize()
```

Comment:
Is `initialize` not called <a href=https://github.com/ParadiseSS13/Paradise/blob/master/code/game/atoms_movable.dm#L26-L28>whenever an `obj` is spawned after round init</a>?